setup.py

```diff
@@ -54,7 +54,7 @@ def get_tag(self):
         "compiler_gym.envs.llvm.service.passes",
         "compiler_gym.envs.llvm.service",
         "compiler_gym.envs.llvm",
-        "compiler_gym.envs.llvm",
```

Comment:
I think this was an accidental redundancy :) (I know it looks like I removed llvm, but line 56 still imports it)